src/common/cmdparse.cc

```diff
@@ -150,23 +150,7 @@ dump_cmd_to_json(Formatter *f, uint64_t features, const string& cmd)
     // name the individual desc object based on the name key
     f->open_object_section(desckv["name"]);
 
-    // Compatibility for pre-nautilus clients that don't know about CephBool
-    std::string val;
-    if (!HAVE_FEATURE(features, SERVER_NAUTILUS)) {
-      auto i = desckv.find("type");
-      if (i != desckv.end() && i->second == "CephBool") {
-        // Instruct legacy clients to send --foo-bar string in place
-        // of a 'true'/'false' value
-        std::ostringstream oss;
-        oss << std::string("--") << desckv["name"];
-        val = oss.str();
-        std::replace(val.begin(), val.end(), '_', '-');
-
-        desckv["type"] = "CephChoices";
-        desckv["strings"] = val;
-      }
-    }
-
+    assert(HAVE_FEATURE(features, SERVER_NAUTILUS));
```

Comment:
Should we keep this one so that pre-nautlius CLI will still work?

Reply:
okay. i am going to drop this change. but i think, unlike the rados clients, ceph CLI should be considered part of in-cluster facility, and the storage admin should have control on its versions

Reply:
@liewegas dropped and repushed.